xs2184-tsw202: Add package to control PoE for Teltonika TSW202 devices

stargieg opened the pull request:
## 📦 Package Details

Patrick Grimm @stargieg 

**Description:**
The xs2184-tsw202 package is used to configure the Power-over-Ethernet controller found on TSW2xx based switches. On many such switches, the PoE doesn't work without this package.

---

## 🧪 Run Testing Details

- **OpenWrt Version:** main
- **OpenWrt Target/Subtarget:** realtek/rtl838x
- **OpenWrt Device:** TSW202

---

## ✅ Formalities

- [ ] I have reviewed the [CONTRIBUTING.md](https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md) file for detailed contributing guidelines.

### If your PR contains a patch:

- [ ] It can be applied using `git am`
- [ ] It has been refreshed to avoid offsets, fuzzes, etc., using
  ```bash
  make package/<your-package>/refresh V=s
  ```
- [ ] It is structured in a way that it is potentially upstreamable
<sub>(e.g., subject line, commit description, etc.)</sub>
<sub>We must try to upstream patches to reduce maintenance burden.</sub>

BKPepe commented on utils/xs2184-tsw202/README.md:
```diff
@@ -0,0 +1,47 @@
+# XS2184 PSE driver and monitor app
```
Readme should be in English.

stargieg replied:
Done

BKPepe reviewed (changes requested):
Based on copyright, I see many files are part of i2c-tools. Why do we need to ship them? How can I know, how much you modified those files? 

Anyway, I am against adding so many files to this repository. It is not going to be reviewed here.

stargieg commented:
Do you have any ideas or plans for how the poe drivers can be standardized?
So that a luci interface can be built.

# packages/utils/poemgr
  SECTION:=utils  
  CATEGORY:=Utilities  
  config /etc/config/poemgr  

# packages/utils/realtek-poe
  SECTION:=net  
  CATEGORY:=Network  
  config /etc/config/poe  

The package depends on the PR https://github.com/openwrt/openwrt/pull/21003

It would be great if someone could take a look at this. The switch is unusable with the OEM firmware for Freifunk.
Thanks for reviewed.

stargieg commented on utils/xs2184-tsw202/README.md:
```diff
@@ -0,0 +1,47 @@
+5. Verwenden Sie eine Pinzette, um den C133-Reset-Kondensator des MS1800K kurzzuschließen und so einen Fehler beim Hochfahren nach dem Einschalten zu simulieren.
+6. Wiederholen Sie Schritt 4 und vergleichen Sie das Verhalten nach dem Ausschalten und erneuten Einschalten mit dem Neustart des MS1800K und den Änderungen des Stromverbrauchs.
+
+## UI einfach
```
Done

stargieg committed:
xs2184-tsw202: add package to control PoE for TSW202 device

this package is used to configure the Power-over-Ethernet

controller xs2184 found on TSW202 switches.

The PoE doesn't work without this package.

Investigation and Dokuments

https://openwrt.org/toh/teltonika/tsw202

Signed-off-by: Patrick Grimm <patrick@lunatiki.de>
